android/src/main/java/com/applicaster/plugin/televisionacademyplayer/ConfigurationRepository.kt

```diff
@@ -15,5 +17,8 @@ object ConfigurationRepository {
         params[CHROMECAST_APP_ID]?.let {
             chromeCastAppId = it.toString()
         }
+        params[HEARTBEAT_INTERVAL]?.let {
```

Comment:
heartbeat interval can be empty string or malformed string.

Reply:
Fixed